TSOClient/FSO.Common.Domain/Realestate/RealestateDomain.cs

```diff
@@ -13,8 +13,11 @@ namespace FSO.Common.Domain.Realestate
 {
     public class RealestateDomain : IRealestateDomain
     {
-        private Regex VALIDATE_NUMERIC = new Regex(".*[0-9]+.*");
-        private Regex VALIDATE_SPECIAL_CHARS = new Regex("[a-z|A-Z|-| |']*");
+        // No need to check redundant regex conditions until you have to throw various errors
+        // (vide tso.client/UI/Panels/UILotPurchaseDialog.cs)
+        // I tried to combine conditions to reduce redundancy
+        private Regex VALIDATE_APOSTROPHES = new Regex("([a-zA-Z -]*)'?(?1)");
```

Comment:
`System.ArgumentException: 'parsing "([a-zA-Z -]*)'?(?1)" - Unrecognized grouping construct.'`

.NET 4.5 does not seem to parse this regex correctly. (.NET regex is somewhat limited) You should verify regex by either testing with C# immediate mode or a simple C# test program.

Reply:
I forgot to test those regexes although I was going to do it… I've changed them, they should be compilable now.